include_core/omrport.h

```diff
@@ -1775,6 +1775,20 @@ typedef struct OMRProcessorDesc {
 #define OMR_FEATURE_X86_XSAVE_AVX512 64 + 2 /* OS Supports AVX-512 (zmm, opmask) state */
 #define OMR_FEATURE_X86_XSAVE_APX    64 + 3 /* OS Supports APX (r16-r31) state */
 
+/* FLAGS FOR AVX 10.X SUPPORT.
+ * These flags are to be set after checking for AVX10 and XCR0 register flags.
+ */
+#define OMR_FEATURE_X86_AVX10_1      64 + 4  /* AVX 10.1 */
+#define OMR_FEATURE_X86_AVX10_2      64 + 5  /* AVX 10.2 */
+
+/* bits 7-15 are reserved */
+
+/* AVX 10 vector length support from CPUID.(EAX=24H, ECX=00H):EBX[16-18] */
+#define OMR_FEATURE_X86_AVX10_128    64 + 16 /* Indicates 128-bit vector support */
+#define OMR_FEATURE_X86_AVX10_256    64 + 17 /* Indicates 256-bit vector support */
+#define OMR_FEATURE_X86_AVX10_512    64 + 18 /* Indicates 512-bit vector support */
```

Comment:
why were bits 7-15 skipped? why didn't we use bits 6, 7, and 8 for these features?